livre/core/Render/GLContext.h

```diff
@@ -43,6 +43,8 @@ class GLContext : public boost::enable_shared_from_this< GLContext >
      */
     void shareContext( GLContextPtr dstContextPtr );
 
+    const GLEWContext* glewGetContext() const;
```

Comment:
Hmm ... Not in this API please, I wanted it boe clean of GlewContext and it is already leaking many places.